test/e2e/data/infrastructure-vsphere/v1.7/bases/cluster-template-topology.yaml

```diff
@@ -0,0 +1,835 @@
+---
```

Comment:
Can we move this file to a "base" folder (instead of bases). Just to keep it more in sync with main

(same for v1.8)